stdlib/3/multiprocessing/managers.pyi

```diff
@@ -2,9 +2,29 @@
 
 # NOTE: These are incomplete!
 
-from typing import Any
+import queue
+import threading
+from typing import Any, Dict, List
+
+class Namespace: ...
+
+_Namespace = Namespace
 
 class BaseManager:
     def register(self, typeid: str, callable: Any = ...) -> None: ...
 
+class SyncManager(BaseManager):
+    def BoundedSemaphore(self, value: Any = ...) -> threading.BoundedSemaphore: ...
+    def Condition(self, lock: Any = ...) -> threading.Condition: ...
+    def Event(self) -> threading.Event: ...
+    def Lock(self) -> threading.Lock: ...
+    def Namespace(self) -> _Namespace: ...
+    def Queue(self, maxsize: int = 0) -> queue.Queue: ...
+    def RLock(self) -> threading.RLock: ...
+    def Semaphore(self, value: Any = ...) -> threading.Semaphore: ...
+    def Array(self, typecode: Any, sequence: Any) -> Any: ...
+    def Value(self, typecode: Any, value: Any) -> Any: ...
+    def dict(self, sequence: Any = ...) -> Dict: ...
```

Comment:
I'm not sure I understand; do you mean something like `Dict[Any, Any]` or `List[Any]`?